packages/tocco-ui/src/EditableValue/typeEditors/StyledDateAbstract.js

```diff
@@ -10,6 +10,9 @@ const StyledDateAbstractControl = styled.div.attrs({
   'data-clear': true
```

Comment:
since you are at it i would move attributes like data-clear out of the styled components. is has nothing to do with styling. instead add those to the elements itself.